core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/sidepanel.jelly

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
   <l:side-panel>
     <l:tasks>
       <l:task href="${rootURL}/" icon="icon-up icon-md" title="${%Back to Dashboard}"/>
-      <l:task href="${rootURL}/manage" icon="icon-gear2 icon-md" permission="${app.ADMINISTER}" title="${%Manage Jenkins}"/>
+      <l:task href="${rootURL}/manage" icon="icon-gear2 icon-md" permission="${app.MANAGE}" title="${%Manage Jenkins}"/>
```

Comment:
This entire view isn't ever used if you don't have ADMINISTER.

Reply:
=> <jenkins-url>/securityRealm/success is reachable by just Overall/Read
So this change makes sense :)